src/renderer/pages/Staking/Operations/common/utils.tsx

```diff
@@ -244,3 +245,9 @@ export const getExplorers = (address: Address, explorers: Explorer[] = []): [Inf
 
   return [{ items: explorersContent }];
 };
+
+export const filterPvRootAccounts = (accounts: Account[], wallets: Wallet[]) => {
+  const walletById = keyBy(wallets, 'id');
```

Comment:
```suggestion
  const walletsMap = dictionary(wallets, 'id', walletUtils.isPolkadotVault);
```
This way you will have `true` or `false` for every wallet 😉